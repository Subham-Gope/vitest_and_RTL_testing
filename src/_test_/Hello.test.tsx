import { render, screen } from "@testing-library/react";

import { describe, expect, test } from "vitest";
import Hello from "../components/Hello";

describe("testing react component", () => {
  test("Hello.tsx", () => {
    const { container } = render(<Hello />);

    expect(container).toMatchSnapshot();

    const heading = screen.getByText("Hello");
    expect(heading).toBeInTheDocument();
  });
});
