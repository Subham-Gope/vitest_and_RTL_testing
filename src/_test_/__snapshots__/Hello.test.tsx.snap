// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`testing react component > Hello.tsx 1`] = `
<div>
  <h1>
    Hello
  </h1>
</div>
`;
